bin/dotd

```diff
@@ -129,7 +130,9 @@ EOS
 user_name = ask_for_name
 
 should_i "morning DTT" do
-  open_pull_request 'test', 'staging', 'DTT (Staging > Test)'
+  GitHub.create_pull_request(
+    base: 'test', head: 'staging', title: 'DTT (Staging > Test)'
+  )
```

Comment:
Though I agree that we want to auto-merge, I wanted to surface this before implementing it. In theory, DOTDs may be looking at the PR before merging it? Do we want to remove this ability?